src/components/Comment/Comment.tsx

```diff
@@ -40,17 +40,21 @@ const commentControls = css`
   align-items: flex-start;
 `;
 
-const commentControlsButton = (pillar: Pillar) => css`
-  ${textSans.xsmall({ fontWeight: "bold" })}
-  margin-right: ${space[2]}px;
-  color: ${palette[pillar][400]};
-  background-color: transparent;
-  border: 0;
-  cursor: pointer;
-  :hover {
-    text-decoration: underline
+const commentControlsButtonWrapper = (pillar: Pillar) => css`
+  button {
+    ${textSans.xsmall({ fontWeight: "bold" })}
+    margin-right: ${space[2]}px;
```

Comment:
We probably shouldn't be applying margins to elements like this. The spacing around an element should, ideally, be context specific

Reply:
yeah it looks like this was unnecessary, good catch